modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,151 @@
+* User/group sharing permission handling: When a share recipient shared a resource with a group the resource owner was a member of (reshare), the resource owner was unable to increase the permissions of the initial share. This has now been fixed. https://github.com/owncloud/core/pull/35884[#35884]
+* User/group sharing permission handling: When a user shared a resource with a group, share recipients (members of the group) were able to remove the share completely (instead of just unsharing from themselves). This has now been fixed. https://github.com/owncloud/core/pull/36120[#36120]
+* External storages now return `StorageNotAvailable` properly on temporary network failures to prevent associated issues (e.g., Desktop clients will not delete local folders anymore when the storage is temporarily not available). https://github.com/owncloud/core/pull/35707[#35707]
+* External storage: Multiple Google Drive external storages can be added again [#34987](https://github.com/owncloud/core/pull/34987
+* The input fields in user administration are not captured by password manager autocompletion anymore. https://github.com/owncloud/core/pull/35931[#35931]
+* Storage encryption with master key in an HSM: Recreating a master key works again https://github.com/owncloud/encryption/pull/128[#128]
+
+=== For developers
+
+* Tech preview for WebDAV Trash bin API (disabled by default) https://github.com/owncloud/core/pull/35716[#35716] https://github.com/owncloud/core/pull/35879[#35879]
+* Tech preview for OCS API for public link share email notifications (disabled by default)https://github.com/owncloud/core/pull/36063[#36063]
+* Tech preview DAV endpoint for public shares (disabled by default) https://github.com/owncloud/core/pull/35932[#35932]
+* Two-factor providers may now display custom challenge messages https://github.com/owncloud/core/issues/34848[#34848]
+* The theming capabilities have been improved by allowing HTML for `Name` and `LogoClaim`. Please check https://github.com/owncloud/theme-example/pull/7/files[the changes to  owncloud/theme-example] if you are interested in making use of this in your theme. https://github.com/owncloud/core/pull/35273[#35273]
+* A new Roles API has been added to allow clients to query the server for available permissions/roles for user/group sharing and public links. In future client releases this endpoint will be used to dynamically display roles/permissions depending on the server's capabilities. https://github.com/owncloud/core/pull/36024[#36024]
+Docs: https://github.com/owncloud/docs/issues/1857
+* A new, improved version of the "Advanced Sharing Permissions" API (v2) has been added to allow ownCloud apps to register additional permissions/restrictions in user/group sharing. Version 1 of the API is still available in parallel. https://github.com/owncloud/core/pull/35836[#35863]
```

Comment:
Maybe add the information that this is a JS Api

Reply:
done